cellxgene_schema_cli/cellxgene_schema/write_labels.py

```diff
@@ -157,8 +157,15 @@ def _get_mapping_dict_feature_id(self, ids: List[str]) -> Dict[str, str]:
         mapping_dict = {}
 
         for i in ids:
-            organism = gencode.get_organism_from_feature_id(i)
-            mapping_dict[i] = get_gene_checker(organism).get_symbol(i)
+            if i == "na":
```

Comment:
i had to build in some logic for making sure a `perturbation_target_gene_id` "na" is mapped to a `perturbation_target_gene_name` "na", as well as handling of comma-delimited cases.

if we productionize this we may want to ensure that the comma-delimited / na cases are only supported for `perturbation_target_gene_id` and not for other columns that use this mapping like `feature_id` -> `feature_name`